apps/frontend/src/pages/ProofExamplePage/index.tsx

```diff
@@ -107,7 +107,6 @@ const certificateInfoTableData: TableRowData<CertificateDto['id']>[] = [{
   id: exampleData.certificate.id ?? '',
   proofId: <EthereumAddress shortify clipboard address={exampleData.certificate.id ?? ''} />,
   product: exampleData.certificate.productType ?? '',
-  beneficiary: exampleData.certificate.beneficiary ?? '',
```

Comment:
I think this should be replaced by `exampleData.beneficiary ` and not removed

Reply:
Totally yeah, forgot to change that. Thanks.

Reply:
Done.